src/tictacclojure/console_ui.clj

```diff
@@ -0,0 +1,28 @@
+(ns tictacclojure.console-ui)
+
+(defn print-out
+  [message]
+  (println message))
+
+(defn- always-validate
+  [user-input]
+  true)
+
+(defn get-input
+  ([]
+    (get-input "" always-validate ""))
+
+  ([prompt]
+    (get-input prompt always-validate ""))
+
+  ([validate error-message]
+   (get-input "" validate error-message))
+
+  ([prompt validate error-message]
+    (if (not-empty prompt) (print-out prompt))
```

Comment:
Will the prompt ever be empty? Unless there is a specific case, I would not worry about it.